src/Tizen.Sensor/Tizen.Sensor/SensorEnumerations.cs

```diff
@@ -319,4 +323,41 @@ public enum DetectorState
         /// <since_tizen> 3 </since_tizen>
         Detected = 1
     }
+    
+    /// <summary>
+    /// The auto rotation state.
+    /// </summary>
+    /// <since_tizen> 3 </since_tizen>
+    public enum AutoRotationState
+    {
+        /// <summary>
+        /// Unknown.
+        /// </summary>
+        /// <since_tizen> 3 </since_tizen>
+        Unknown = -1,
+
+        /// <summary>
+        /// DEGREE_0 sate.
+        /// </summary>
+        /// <since_tizen> 3 </since_tizen>
+        DEGREE_0 = 0,
```

Comment:
```suggestion
        Degree_0 = 0,
```

https://github.com/Samsung/TizenFX/blob/39f533cdf36c04419760bc08683f5db979642528/src/ElmSharp/ElmSharp/Window.cs#L33

Reply:
@myroot  done! :smile: